html/getstarted.html

```diff
@@ -0,0 +1,485 @@
+
+			<nav>
+				<a href="index.html">Home</a>
+				<a href="getstarted.html">Quick Start</a>
+				<a href="docs.html">Documentation</a>
+				<a href="community.html">Community</a>
+				<a href="faq.html">FAQ</a>
+				<a href="https://github.com/mlpack/mlpack">GitHub</a>
+			</nav>
+		</header>
+	</div>
+
+	<div class="box a"></div>
+</div>
+<!-- NAVIGATION END -->
+
+<br>
+
+<!-- QUICKSTART START -->
+<center style="padding-top:40px;">
+	<div style="max-width:800px">
+
+		<!-- SUB-NAVIGATION START -->
+		<div class="nav-element" id="sub-nav">
+			<p>
+				<span style="font-weight:600">Quickstart</span><br>
+				<span class="nav-sub" onclick="ScrollTo('page-assets')">Get your assets</span><br>
+				<span class="nav-sub" onclick="ScrollTo('page-compiling')">Compiling manually</span><br>
+				<span class="nav-sub" style="padding-left:16px" onclick="ScrollTo('page-compiling-linux')">Linux</span><br>
+				<span class="nav-sub" style="padding-left:16px" onclick="ScrollTo('page-compiling-windows')">Windows</span><br>
+			</p>
+		</div>
+		<!-- SUB-NAVIGATION END -->
+
+		<div id="helper"></div>
+
+		<h3 style="text-align:left">Quick Start</h3>
+		<p align="justify">
+			There are multiple ways to get mlpack up and running. Python bindings can be
+			installed using pip or conda, or built from source (see <a id="built-source" href="https://www.mlpack.org/doc/mlpack-3.4.1/doxygen/build.html">Build from
+			source tutorials</a>). Julia bindings can be installed via Julia's 'Pkg' package manager. For C++, if mlpack is not available via your preferred
+			OS package manager, or if you need to build your own version (e.g. to apply
+			optimizations, use a different set of BLAS/LAPACK, or build a different
+			configuration), please refer to the <a id="built-manually" href="https://www.mlpack.org/doc/mlpack-3.4.1/doxygen/build.html">Build from
+			source</a> tutorials. For Windows, prebuilt binaries will help you get started
+			without the need of building mlpack. These packages include both the C++
+			mlpack library as well as the CLI tools.
+		</p>
+
+		<p align="justify">
+				Once you get mlpack running, check out the <a
+				href="https://www.mlpack.org/docs.html">documentation</a> or the <a
+				href="https://github.com/mlpack/examples/">examples repository</a>, which
+				contains simple example usages of mlpack.
+		</p>
+	</div>
+</center>
+<!-- QUICKSTART END -->
+
+<br>
+
+<!-- QUICKSTART-CLOUD START -->
+<div class="grid-simple-package">
+	<div></div>
+
+	<div style="width:200px;height:40px;border-radius:5px;background-color:#BABFC6">
+		<div style="padding-top:10px"><a style="text-decoration:none;color:black" href="http://lab.mlpack.org/v2/gh/mlpack/examples/master?urlpath=lab">Jupyter cloud</a></div>
+	</div>
+
+	<div style="width:200px;height:40px;border-radius:5px;background-color:#BABFC6">
+		<div style="padding-top:10px"><a style="text-decoration:none;color:black" href="">Docker Linux</a></div>
```

Comment:
Should this link somewhere?

Reply:
Yes, fixed in https://github.com/mlpack/mlpack.org/pull/26/commits/9cce2b86474a9815583a438425e0b6267b142967.